src/ipc-handlers.ts

```diff
@@ -189,6 +190,8 @@ export async function createSite(
 
 		if ( ! ( await pathExists( nodePath.join( path, 'wp-config.php' ) ) ) ) {
 			await installSqliteIntegration( path );
+		} else {
+			await updateSiteUrlToLocal( details.id );
```

Comment:
I was thinking about different options and places to do it, but this one looks as the most correct one, since existence of `wp-config.php` tells us - was the site already initialized or not.